code/controllers/master.dm

```diff
@@ -32,6 +32,8 @@ GLOBAL_REAL(Master, /datum/controller/master) = new
 
 	/// world.time of last fire, for tracking lag outside of the mc
 	var/last_run
+	/// the last MAPTICK_LAST_INTERNAL_TICK_USAGE the mc recorded, used for tracking tick portions outside the MC.
+	var/last_maptick
```

Comment:
this should always be 0, no?

Reply:
what?

Reply:
idk for some reason I thought world.map_cpu reset at the start of the tick
seems weird to store like this regardless tho idk

Reply:
It's used to check if a given proc is executing after maptick but honestly it's not needed

Reply:
Also it's not reset

Reply:
ye I uh, I realized that once I thought about it for more then 2 seconds